packages/cw721-init-msg-registry/src/contracts/mod.rs

```diff
@@ -0,0 +1,13 @@
+#[cfg(feature = "cw721-base")]
+pub mod cw721;
+
+#[cfg(feature = "sg721-base")]
+pub mod sg721;
+
+cfg_if::cfg_if! {
+  if #[cfg(feature = "sg721-base")] {
```

Comment:
What does this mean, in case another custom cw721-contract should be used? Whole repo needs to be forked, make changes here in `cw721-init-msg-registry` package and adjust in bridge-contract?